DetourCrowd/Source/DetourCrowd.cpp

```diff
@@ -386,7 +386,7 @@ bool dtCrowd::init(const int maxAgents, const float maxAgentRadius, dtNavMesh* n
 	m_maxAgents = maxAgents;
 	m_maxAgentRadius = maxAgentRadius;
 
-	dtVset(m_ext, m_maxAgentRadius*2.0f,m_maxAgentRadius*1.5f,m_maxAgentRadius*2.0f);
+	dtVset(m_halfExtents, m_maxAgentRadius*2.0f,m_maxAgentRadius*1.5f,m_maxAgentRadius*2.0f);
```

Comment:
It is used during normal operation too, not only if it detects that it needs recovering, so that might be a little misleading. I think Mikko's suggestion, maybe with a comment on the field that it is chosen large for recovery reasons, is best.